modules/oe_editorial_entity_version/src/Form/CorporateWorkflowEntityRevisionRevertForm.php

```diff
@@ -97,7 +98,7 @@ public static function create(ContainerInterface $container) {
    * {@inheritdoc}
    */
   public function getFormId() {
-    return 'node_version_restore_confirm';
+    return 'entity_version_restore_confirm';
```

Comment:
We should have a test covering link lists similar to NodeVersionRestoreTest.

Reply:
I think it's ok, the functionality is tested with an entity type